qiskit/providers/models/__init__.py

```diff
@@ -50,3 +51,13 @@
 from .backendstatus import BackendStatus
 from .jobstatus import JobStatus
 from .pulsedefaults import PulseDefaults, Command
+
+
+warnings.warn(
+    "qiskit.providers.models is deprecated since Qiskit 1.2 and will be removed in Qiskit 2.0."
+    "With the removal of Qobj, there is no need for these schema-conformant objects. If you still need"
+    "to use them, it could be because you are using a BackendV1, which it is also deprecated in favor"
+    "of BackendV2",
```

Comment:
```suggestion
    "qiskit.providers.models is deprecated since Qiskit 1.2 and will be removed in Qiskit 2.0."
    "With the removal of Qobj, there is no need for these schema-conformant objects. If you still need"
    "to use them, it could be because you are using a BackendV1, which is also deprecated in favor"
    "of BackendV2",
```

Reply:
Is there test asserting that this deprecation warning is raised? I might be wrong but I have the impression that it will always be "overwritten" by the specific class deprecation warnings (which I am also not against).